### PR TITLE
3DMLoader: Clean up

### PR DESCRIPTION
**Pull request by LeviPesin**
Related issue: -

**Description**

Clean up.

### CODE REVIEW

File: examples/jsm/loaders/3DMLoader.js [hunk 1]
```diff
@@ -181,26 +182,13 @@ class Rhino3dmLoader extends Loader {
...
 	_compareMaterials( material ) {
 
-		const mat = {};
-		mat.name = material.name;
-		mat.color = {};
-		mat.color.r = material.color.r;
-		mat.color.g = material.color.g;
-		mat.color.b = material.color.b;
-		mat.type = material.type;
+		const json = JSON.stringify( material );
```
Comment (Mugen87) [inline examples/jsm/loaders/3DMLoader.js:185]: To get this PR merged, can you please revert the changes related to how materials are compared? Without the feedback of the original dev, I don't like to change this section.

File: examples/jsm/loaders/3DMLoader.js [hunk 2]
```diff
@@ -418,7 +406,7 @@ class Rhino3dmLoader extends Loader {
 					const xf = iRef.geometry.xform.array;
 
 					const matrix = new Matrix4();
-          			matrix.set( xf[ 0 ], xf[ 1 ], xf[ 2 ], xf[ 3 ], xf[ 4 ], xf[ 5 ], xf[ 6 ], xf[ 7 ], xf[ 8 ], xf[ 9 ], xf[ 10 ], xf[ 11 ], xf[ 12 ], xf[ 13 ], xf[ 14 ], xf[ 15 ] );
+					matrix.elements = xf;
```
Comment (Mugen87) [inline examples/jsm/loaders/3DMLoader.js:409]: Please avoid to change property references like that. If some code has a reference on the existing `elements` array, assigning a new array elsewhere will break it.

**Comment by fraguada** [comment]
Apologies that I missed this ping. I will be able to review these changes next week.

File: examples/jsm/loaders/3DMLoader.js (hunk 1 above)
Comment (LeviPesin) [reply examples/jsm/loaders/3DMLoader.js]: Done.

File: examples/jsm/loaders/3DMLoader.js (hunk 2 above)
Comment (LeviPesin) [reply examples/jsm/loaders/3DMLoader.js]: Fixed!

**Comment by Mugen87** [comment]
Merging for now since the changes look save.
